### PR TITLE
Added permission tag requires use_camera permission.

### PR DESCRIPTION
**Pull request by tohman**

### CODE REVIEW

File: api/blackberry_media_camera.js [hunk 1]
```diff
@@ -20,6 +20,7 @@
  * <p>The methods are static and don't required to create an instance of a camera.</p>
  * @toc {Media} Camera
  * @featureID blackberry.media.camera
+ * @permission use_camera This permission should be granted in order to be able to invoke native camera.
```
Comment (jeffheifetz) [inline api/blackberry_media_camera.js]: How about

```
@permission use_camera Permits the user to invoke the native camera application.
```

Comment (tohman) [inline api/blackberry_media_camera.js]: The change is coming..

File: api/blackberry_media_camera.js [hunk 2]
```diff
@@ -20,6 +20,8 @@
  * <p>The methods are static and don't required to create an instance of a camera.</p>
  * @toc {Media} Camera
  * @featureID blackberry.media.camera
+ * @permission use_camera Permits the user to invoke the native camera application.  
+ * @permission [access_shared] Permits the user to retrieve a path to the file stored in the system.
```
Comment (jeffheifetz) [inline api/blackberry_media_camera.js]: I know I just bugged you about the permissions, but do these sound clearer. I no longer like user and I think shorter is better based on how they currently display.

What do you think?

```
@permission use_camera Permits your app to use the camera.  
@permission [access_shared] Permits your app to access the shared files and folders.
```

Comment (tohman) [inline api/blackberry_media_camera.js]: Like the first one.
Second permission is needed in the case user want to listen for the onCapture event and interested in its filePath parameter, for different reasons i.e. saving, storing, displaying, not necessarily to access the file itself.

Comment (jeffheifetz) [inline api/blackberry_media_camera.js]: Doesn't that just mean it permits read/write access to the shared folder...What you do with it is up to the developer, right?

**Comment by tohman** [comment]
To retrieve the full path to file saved there is a need in action script to analyze shared folder and for that a permission required.
It shouldn't be optional since onCaptured to whom filePath is callback is passed is a required parameter.

**Comment by jeffheifetz** [comment]
Great, this looks ready to merge

**Pull request merged**